deploy/dev/process-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: "0.5"
+
+processes:
+  caddy:
+    command: caddy run --config ./caddy/Caddyfile
+  clients:
+    command: cd ../../clients/spas && npm run dev
+  identity:
+    command: cd ../../services/identity && npm run start
+  notes:
+    command: cd ../../services/notes/notes-service && npm run start
+
+  # Use envsubst to have prometheus config based on env variables
+  prometheus:
+    command: sh -c 'envsubst < ./prometheus/prometheus.yml > /tmp/prometheus.yml && prometheus --config.file=/tmp/prometheus.yml --storage.tsdb.path=/tmp/prometheus --web.listen-address="${PROMETHEUS_HOST}"'
+
+  # Find nix install to set homepath
+  grafana:
+    command: sh -c 'GRAFANA_BIN=$(which grafana); GRAFANA_STORE=$(dirname $(dirname "$GRAFANA_BIN")); GF_SERVER_HTTP_ADDR=$${GRAFANA_HOST%%:*} GF_SERVER_HTTP_PORT=$${GRAFANA_HOST##*:} grafana server --homepath="$GRAFANA_STORE/share/grafana"'
+    environment:
+      - GRAFANA_HOST=${GRAFANA_HOST:-localhost:3000}
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_AUTH_DISABLE_LOGIN_FORM=true
+
+      # Need to provide paths; default paths are read-only
+      - GF_PATHS_DATA=/tmp/grafana
+      - GF_PATHS_LOGS=/tmp/grafana/logs
+      - GF_PATHS_PLUGINS=/tmp/grafana/plugins
+      
+  loki:
+    command: loki -config.file=./loki/loki-config.yaml -server.http-listen-address="${LOKI_HOSTNAME}" -server.http-listen-port="${LOKI_PORT}"
+    
+  node_exporter:
+    command: sh -c 'node_exporter --path.procfs=/proc --path.sysfs=/sys --collector.filesystem.mount-points-exclude='\''^/(sys|proc|dev|host|etc)($|/)'\'' --web.listen-address="${NODE_EXPORTER_HOST}"'
```

Comment:
These commands were generated, then simplified. They seem mostly reasonable to me but let me know if there are changes you'd recommend! Certainly this is simpler than docker-compose... though I do wonder how well they work on not-a-mac.